src/UserSpaceInstrumentation/TestUtils.cpp

```diff
@@ -0,0 +1,97 @@
+#include "TestUtils.h"
+
+#include <absl/strings/str_cat.h>
+#include <capstone/capstone.h>
+
+#include "ObjectUtils/Address.h"
+#include "ObjectUtils/ElfFile.h"
+#include "ObjectUtils/LinuxMap.h"
+#include "OrbitBase/ExecutablePath.h"
+#include "OrbitBase/Logging.h"
+#include "OrbitBase/UniqueResource.h"
+
+namespace orbit_user_space_instrumentation {
+
+AddressRange GetFunctionAddressRangeInMemoryOrDie(std::string_view function_name) {
+  auto modules = orbit_object_utils::ReadModules(getpid());
+  CHECK(!modules.has_error());
+  std::string module_file_path;
+  AddressRange address_range_code(0, 0);
+  for (const auto& m : modules.value()) {
+    if (m.name() == orbit_base::GetExecutablePath().filename()) {
```

Comment:
this assumes soname is the same as a filename. maybe then explicitly compare m.file_path()?